plugins/github-deployments/src/components/GithubDeploymentsCard.tsx

```diff
@@ -76,6 +86,7 @@ export const GithubDeploymentsCard = ({
         entity?.metadata.annotations?.[GITHUB_PROJECT_SLUG_ANNOTATION] || ''
       }
       last={last || 10}
+      locations={locations}
```

Comment:
Feels like it should only accept a single optional string -  if the source annotation is present it should be used for sure. So we could essentially do

```ts
  const location: string | undefined = [
    entity?.metadata.annotations?.[SOURCE_LOCATION_ANNOTATION],
    entity?.metadata.annotations?.[LOCATION_ANNOTATION],
  ].filter(Boolean)[0];
```

and pass that in